test/cert-ceremonies/intermediate-ceremony-rsa.yaml

```diff
@@ -16,12 +16,10 @@ certificate-profile:
     country: US
     not-before: 2020-01-01 12:00:00
     not-after: 2040-01-01 12:00:00
-    crl-url:  http://example.com/crl
-    issuer-url:  http://example.com/root
+    crl-url:  http://rsa.root.ca/crl
```

Comment:
Why the change in URL here?

Reply:
Sounds good. Can we continue having them be subdomains of `example.com` though? `root.ca` is a real domain name.

Reply:
Ahahah I totally forgot that Canada is a real place.